project.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Portfolio Project</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="portfolio.html">Home</a></li>
+			<li><a href="portfolio.html#portfolio">Portfolio</a></li>
+			<li><a href="portfolio.html#aboutme">About Me</a></li>
+			<li><a href="portfolio.html#contact">Contact</a></li>
+		 </ul>
+		 <!-- <div><img src="" alt=""/> Place for logo -->
+	</nav>
+
+	<div class="container project">
+		<div class="projectimg"><img src="projectimg.jpg" alt="Photos of Project" width="600"></div>
```

Comment:
dont put widths on images in html, use CSS

Reply:
I'm going to redo this whole page, the layout doesn't work.